mesh/amqp.c

```diff
@@ -610,6 +630,11 @@ static void control_message_handler(struct amqp_thread_context *context)
 							msg.publish.size);
 			return;
 
+		case SUBSCRIBE:
```

Comment:
Shouldn't we call amqp_subscribe_topics() after updating the list?

Also, what if topics are duplicated?

Reply:
Current API requires to call methods in right order and doesn't allow for modifications during run time (Everything must be setup during startup).  Personally I don't like that and I want to change it, but in fact it do everything what we need and currently there is no case to change it.

Reply:
Ok, you don't like and I don't really like it, let's fix it then.

Let's assume that area list can change *without* application restart, and let's provide API that works with that assumption, even if it's not true right now.